src/main/java/seedu/moneygowhere/logic/commands/reminder/AddReminderCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.moneygowhere.logic.commands.reminder;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.moneygowhere.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.moneygowhere.logic.parser.CliSyntax.PREFIX_MESSAGE;
+
+import seedu.moneygowhere.logic.commands.CommandResult;
+import seedu.moneygowhere.logic.commands.ReminderCommand;
+import seedu.moneygowhere.logic.commands.exceptions.CommandException;
+import seedu.moneygowhere.model.Model;
+import seedu.moneygowhere.model.reminder.Reminder;
+
+/**
+ * Adds a Reminder to the Reminder list.
+ */
+public class AddReminderCommand extends ReminderCommand {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Reminder to MoneyGoWhere. "
+            + "Parameters: "
+            + PREFIX_DATE + "DATE "
+            + PREFIX_MESSAGE + "MESSAGE ";
+
+    public static final String MESSAGE_SUCCESS = "New Reminder added: %1$s";
+
+    private final Reminder toAdd;
```

Comment:
Better to keep it as `reminder` or `value`

Reply:
i will change the variable name to reminderToAdd.